### PR TITLE
community/events: Update CFP details for Ceph Days London 2024

### PR DESCRIPTION
**Pull request by gauravsitlani**

### CODE REVIEW

**Comment by gauravsitlani** [comment]
Hello @neha-ojha @dabukalam @oritwas can you please review and share your feedback if it looks good?

File: src/en/community/events/2024/ceph-days-london/index.md [hunk 1]
```diff
@@ -30,10 +31,10 @@ The registration will be limited!
 
 ## Important Dates
 
-- **CFP Opens:** TBD
-- **CFP Closes:** TBD
-- **Speakers receive confirmation of acceptance:** TBD
-- **Schedule Announcement:** TBD
+- **CFP Opens:** 2024-06-10
+- **CFP Closes:** 2024-07-30
```
Comment (dabukalam) [inline src/en/community/events/2024/ceph-days-london/index.md:35]:
```suggestion
- **CFP Closes:** 2024-06-30
```

File: src/en/community/events/2024/ceph-days-london/index.md [hunk 2]
```diff
@@ -9,14 +9,15 @@ sponsors:
...
+        logo: /assets/bitmaps/logo-ibm.png
 tags:
   - ceph days
 ---
 
 ### Bringing Ceph to London
 
 <a class="button" href="https://ti.to/open-source-events/ceph-days-london-2024">Register Now!</a>
-<a class="button" href="#">CFP Opens Soon</a>
+<a class="button" href="https://docs.google.com/forms/d/e/1FAIpQLSeCWMptX-MKyj2uc4gco5sIewICymKFY1EuSddzaXmiX28m0w/viewform?usp=sf_link">CFP Open!</a>
```
Comment (dabukalam) [inline src/en/community/events/2024/ceph-days-london/index.md:20]:
```suggestion
<a class="button" href="https://docs.google.com/forms/d/e/1FAIpQLSeCWMptX-MKyj2uc4gco5sIewICymKFY1EuSddzaXmiX28m0w/viewform?usp=sf_link">Submit a Talk!</a>
```

**Review by dabukalam — COMMENTED** [review]
Couple of suggestions, otherwise LGTM

**Review by dabukalam — APPROVED** [review]

**Pull request merged**